danieldare/ireporter-react
# [WIP] - React-Redux initial setup

**What does this PR do?**
- Adds redux
- Adds the dashboard, Login , Register,  Landing page, create Incident functionality


**Description of Task to be completed?**
- set up react
- set up redux
- login functionality
- register functionality
- create incident
- create dashboard UI
- create login and register UI
- create landing page UI

**How should this be manually tested?**
- Check out this branch
- Run `npm i`
- Run `npm run start`

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**


**Screenshots (if appropriate)**
